lib/shopify-cli/partners_api/organizations.rb

```diff
@@ -4,23 +4,23 @@ class Organizations
       class << self
         def fetch_all(ctx)
           resp = PartnersAPI.query(ctx, "all_organizations")
-          (resp.dig("data", "organizations", "nodes") || []).map do |org|
+          (resp&.dig("data", "organizations", "nodes") || []).map do |org|
```

Comment:
does this mean if the response is nil this block won't run? doesn't that result in the next call failing on a nil error...?

Reply:
https://mitrev.net/ruby/2015/11/13/the-operator-in-ruby/ Its the safe nav operator!